### PR TITLE
Fix: Upcoming dialog box in TrackSessionsActivity

### LINKED ISSUE #2248: Upcoming function is not working in tracks section
**Actual Behaviour**

while selecting any track upcoming track is not working.

**Expected Behaviour**

I think it should be displayed as it is shown in location section.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by shivaraju07**
Fixes #2248

### CODE REVIEW

**Comment by iamareebjamal** [comment]
@srv-twry @mayank8318 @nikit19 @simarsingh24 Please help review PRs of each other so we can wrap up things quickly

**Comment by nikit19** [comment]
Sure 👍

**Comment by simarsingh24** [comment]
@shivaraju07 can you attach some screenshots after the change.

**Comment by shivaraju07** [comment]
After the changes:
media-20180128

File: android/app/src/main/java/org/fossasia/openevent/activities/TrackSessionsActivity.java [hunk 1]
```diff
@@ -67,6 +73,13 @@
     private SearchView searchView;
     private Menu menu;
 
+    private Dialog upcomingDialogBox;
+    private ImageView trackImageIcon;
+    private TextView upcomingSessionText;
+    private TextView upcomingSessionTitle;
```
Comment (mayank8318) [inline android/app/src/main/java/org/fossasia/openevent/activities/TrackSessionsActivity.java:79]: Do we need all these global variables?

**Comment by mayank8318** [comment]
Also IMO a snackbar for no upcoming events is better than an empty dialog box.

**Comment by dr0pdb** [comment]
> Also IMO a snackbar for no upcoming events is better than an empty dialog box.

Yes, but for now let's stick to the dialog box.

**Comment by iamareebjamal** [comment]
Please add a padding on top of textview. It should appear in center

File: android/app/src/main/java/org/fossasia/openevent/activities/TrackSessionsActivity.java (hunk 1 above)
Comment (shivaraju07) [reply android/app/src/main/java/org/fossasia/openevent/activities/TrackSessionsActivity.java:79]: Yes, these are used to describe the title and message. I think it should be there.

File: android/app/src/main/java/org/fossasia/openevent/activities/TrackSessionsActivity.java [hunk 2]
```diff
@@ -164,6 +179,46 @@ private void loadSessions() {
         });
     }
 
+
```
Comment (simarsingh24) [inline android/app/src/main/java/org/fossasia/openevent/activities/TrackSessionsActivity.java]: please remove this space

**Comment by dr0pdb** [comment]
LGTM.
Just add a final screenshot of what it looks like now. Also update your branch with the latest code.

Side note: Try to follow the PR and commit naming conventions. 
It would be better to change the title and commit message to ```fix: Upcoming dialog box in TrackSessionsActivity```
Also check for typos eg. Upcomming -> Upcoming.

**Comment by shivaraju07** [comment]
@srv-twry sure. I will keep it in mind.

ScreenShot:
new

**Comment by shivaraju07** [comment]
@srv-twry  shall I update the branch?

**Comment by dr0pdb** [comment]
> shall I update the branch?

Yes, please

> sure. I will keep it in mind.

:+1: , While updating you can also change the commit message. Also change the title of this PR.

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**